examples/ziti-end-to-end/README.md

```diff
@@ -0,0 +1,172 @@
+# Setting up an End-to-End Ziti network
+## Prerequisites:
+
+Go 1.19 installed and setup: [download](https://go.dev/doc/install). You're going to want to set you `$GOPATH` env variable which can be anywhere on your system. This is where modules, packages, and installed binaries will end up. Putting `$GOPATH/bin` in your path will help with running the executables later.
+
+Ziti Binaries and relevant configs.
+
+[OpenZiti](https://github.com/openziti/ziti/tree/main) contains downloads for the latest builds or you can build it yourself. There are many example configs [in the etc dir](https://github.com/openziti/ziti/tree/main/etc).
+
+I have set up helpful environment variable for how I chose to get setup. I cloned the Ziti repository for local builds and set `$ZITI_SOURCE` to that directory. I then went into that directory and ran 
+```
+go install ./...
+```
+
+This will output the binary to `$GOPATH/bin`. I then do the same thing with this repository to install the go-httpbin binaries
+
+## Starting up the Ziti Network:
+
+### Controller Setup
+
+First we run the ziti controller with the arguent being which ever config file we chose. For this example I am using the `ctrl.with.edge.yml` file bundled for examples in the ziti repository.
+```
+ziti-controller run --log-formatter pfxlog $ZITI_SOURCE/ziti/etc/ctrl.with.edge.yml
```

Comment:
The way this is written is sounds like `ZITI_SOURCE` is the top-level of the main "ziti" repo checked out working copy, but if so then the paths to sub-dir `etc` are incorrect because of the extra `/ziti` in the path.

With that extra `/ziti` in the path the correct value of `ZITI_SOURCE` would be the parent dir of the "ziti" repo working copy, so we just need to clarify the correct value or re-write the paths to omit the extra `/ziti` in `$ZITI_SOURCE/ziti`.

Reply:
Looks like we need to assign `ZITI_DATA` too. It'll be important for the reader to `export ZITI_SOURCE ZITI_DATA` after assignment so they're available to child procs like `ziti-controller` which is reading the config file where the vars are interpolated.

Reply:
Good catch! Didn't notice that. Removed the extra `/ziti`'s

Reply:
@camotts I took another look at this today because I'm comparing the steps in this article with the local-dev guide in the main Ziti repo. I noticed that the variables in the controller and router configs are like this:

```
${ZITI_SOURCE}/ziti/etc/ca/intermediate/certs/ctrl-client.cert.pem
```
So, the value of `ZITI_SOURCE` must be the parent dir of this repo, and the main `ziti` repo needs to be checked out in an adjacent sibling dir.

This means a valid value if `export ZITI_SOURCE=..` if PWD is the checked out copy of either repo.

I'm not sure why we're using those values in the controller and router configs.